Amazon.Ion.ObjectMapper/IonSerializer.cs

```diff
@@ -173,6 +173,8 @@ public class IonSerializationOptions
         public readonly bool PermissiveMode;
         
         public Dictionary<Type, IIonSerializer> IonSerializers { get; init; }
+
+        public Dictionary<string, IIonSerializer> AnnotatedIonSerializers { get; init; }

```

Comment:
This shouldn't be present - we do not want to add to the public API unless specified. Can you link to where in the spec this says it's needed?